cirq-google/cirq_google/engine/virtual_engine_factory.py

```diff
@@ -124,6 +124,19 @@ def create_noiseless_virtual_engine_from_device(
     return SimulatedLocalEngine([_create_virtual_processor_from_device(processor_id, device)])
 
 
+def create_noiseless_virtual_processor_from_proto(
+    processor_id: str,
+    device_specification: v2.device_pb2.DeviceSpecification,
+    gate_sets: Optional[Iterable[serializable_gate_set.SerializableGateSet]] = None,
+) -> SimulatedLocalProcessor:
```

Comment:
Missing docstring.  Also be aware that this will need to change once @verult finishes the device spec refactor.

Reply:
done; and pluralized `create_noiseless_virtual_engine_from_proto` docstring args for consistency (and to contrast with this new docstring)